src/App.tsx

```diff
@@ -1,9 +1,22 @@
 import { useState, useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import axios, { AxiosResponse } from 'axios';
+import { v4 as uuidv4 } from 'uuid';
+import { Alert, Grow } from '@mui/material';
 import { queryURL, attributesURL, isFederationAPI, nodesURL } from './utils/constants';
-import { RetrievedAttributeOption, AttributeOption, NodeOption, Result } from './utils/types';
+import {
+  RetrievedAttributeOption,
+  AttributeOption,
+  NodeOption,
+  FieldInput,
+  FieldInputOption,
+  Result,
+} from './utils/types';
 import QueryForm from './components/QueryForm';
 import ResultContainer from './components/ResultContainer';
+import Navbar from './components/Navbar';
+import { useSnackStack } from './components/SnackStackProvider';
+import SnackStack from './components/SnackStack';
 import './App.css';
```

Comment:
Commenting here: This is a really impressive amount of work to get the stacking snackbar to work. But now that's our code to keep alive and I would very much avoid that. Essentially

- MUI / Android says: do not stack toasts
- but we want to do it anyway, breaking MUI / Android visual spec
- MUI says: good luck, you're on your own. But you should know that if you do this, here is a third party library that handles it for you: https://mui.com/material-ui/react-snackbar/#third-party-integrations
- we say: thanks a lot, but we'll both break with MUI and not use the third-party app

That last point is where I disagree. I think we should either

- stop stacking toasts (not good because toasts are nice) OR
- just use notistack: https://notistack.com/features/basic and don't bother adding all this stuff ourselves

Reply:
I agree let's use notistack.